source/npm/util.js

```diff
@@ -85,8 +85,9 @@ exports.version = () => execa.stdout('npm', ['--version']);
 
 exports.verifyRecentNpmVersion = async () => {
 	const npmVersion = await exports.version();
+	const requiredNpmVersion = require('../../package.json').engines.npm;
 
-	if (version(npmVersion).satisfies('<6.8.0')) {
-		throw new Error('Please upgrade to npm@6.8.0 or newer');
+	if (!version(npmVersion).satisfies(requiredNpmVersion)) {
```

Comment:
You could add a helper function to simplify this:

```js
const versionSatisfiesRequirement = (dependency, v) => {
    const depRange = require('../../package.json').engines[dependency];
    return version(v).satisfies(depRange);
};

if (!versionSatisfiesRequirement('npm', npmVersion) {
    // npm version error
}

if (!versionSatisfiesRequirement('git', npmVersion) {
    // git version error
}

if (!versionSatisfiesRequirement('yarn', npmVersion) {
    // yarn version error
}
```

Reply:
Also, [`read-pkg-up`](https://github.com/sindresorhus/read-pkg-up) should be used here instead of `require('../../package.json')`.